app/jitsi/models.py

```diff
@@ -0,0 +1,119 @@
+from . import db
+from datetime import datetime
+from sqlalchemy_serializer import SerializerMixin
+from sqlalchemy.ext.hybrid import hybrid_property
+
+USER_TIMEOUT = 30
+
+class User(db.Model, SerializerMixin):
+    __tablename__ = 'users'
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(100))
+    avatar = db.Column(db.String())
+    last_seen = db.Column(db.Integer, default=lambda: datetime.utcnow().timestamp())
+
+    @hybrid_property
+    def is_active(self):
+         return self.last_seen > datetime.utcnow().timestamp() - USER_TIMEOUT
+
+    def ping(self):
+        self.last_seen = datetime.utcnow().timestamp()
+        db.session.add(self)
+        db.session.commit()
+
+    @staticmethod
+    def create(username, avatar):
+        # TODO change this once Daniel fixes avatars
+        avatar = '-'.join(map(str, avatar))
+        user = User(username=username, avatar=avatar)
+        db.session.add(user)
+        db.session.commit()
+        return user
+
+    @staticmethod
```

Comment:
these should probably be class methods so you aren't constantly typing the class name

Reply:
These don't operate on a user instance, though!